android/runtime/v8/src/native/TypeConverter.cpp

```diff
@@ -271,6 +274,50 @@ Local<Function> TypeConverter::javaObjectToJsFunction(Isolate* isolate, JNIEnv *
 	return persistentV8Object.Get(isolate);
 }
 
+jobject TypeConverter::jsObjectToJavaPromise(Isolate* isolate, Local<Object> jsObject)
+{
+	JNIEnv *env = JNIScope::getEnv();
+	if (!env) {
+		return NULL;
+	}
+	return TypeConverter::jsObjectToJavaPromise(isolate, env, jsObject);
+}
+
+jobject TypeConverter::jsObjectToJavaPromise(Isolate* isolate, JNIEnv *env, Local<Object> jsObject)
+{
+    Local<Promise::Resolver> resolver = jsObject.As<Promise::Resolver>();
+    Persistent<Promise::Resolver, CopyablePersistentTraits<Promise::Resolver>> persistent(isolate, resolver);
+    persistent.MarkIndependent();
```

Comment:
`MarkIndependent()` has been removed from V8, we only added it back for compatibility with older SDKs. This line can safely be removed, the persistent object won't be GC'ed anyway (unless `SetWeak()` is called)